src/lib/blocks.js

```diff
@@ -100,6 +100,14 @@ export default function (vm) {
         this.jsonInit(json);
     };
 
+    ScratchBlocks.Blocks.motion_glideto_menu.init = function () {
```

Comment:
I'm not all that experienced with the scratch-blocks/-gui/etc codebase, but I'd make the guess that this causes an error because `ScratchBlocks.Blocks.motion_glideto_menu` isn't defined. You can see that, in scratch-blocks, `motion_goto`, `sensing_of_object_menu`, etc are [all defined](https://github.com/LLK/scratch-blocks/blob/829560a0dff52ffebcfb5927a85fa9182d91a054/blocks_vertical/motion.js#L169); but `motion_glideto_menu` isn't.

You've already made a PR which [addresses this](https://github.com/LLK/scratch-blocks/pull/1046/files#diff-11acb9531abd6af6a47ec6908d0d2dc8R268); that's LLK/scratch-blocks#1046.

I guess what you need to do is make Travis understand that this PR *depends* on that one. I just don't know how to do that (perhaps this is the reason thisandagain pinged rschamp, who might actually know) :)